automation/src/test/scala/org/broadinstitute/dsde/test/SamConfig.scala

```diff
@@ -1,10 +1,11 @@
-package org.broadinstitute.dsde.test
+package org.broadinstitute.dsde.workbench.test
 
 import org.broadinstitute.dsde.workbench.config.CommonConfig
 
 object SamConfig extends CommonConfig {
   // from common: qaEmail, pathToQAPem
   object GCS extends CommonGCS {
     val appsDomain = gcsConfig.getString("appsDomain")
+    val pathToSamTestFirestoreAccountPath = gcsConfig.getString("firestoreAccountPath")
```

Comment:
where is this populated? In our unit tests do we use a real account talking to real firestore? Is there any way to mock that out? Is there a local firestore emulator?

https://firebase.google.com/docs/firestore/security/test-rules-emulator#install_the_emulator

Reply:
it's populated https://github.com/broadinstitute/firecloud-automated-testing/blob/master/configs/sam/application.conf.ctmpl#L47.

I wasn't aware there's emulator. I'll check it out. But I guess that can be in another PR?